app/_src/deck/reference/deck_file_kong2tf.md

```diff
@@ -783,7 +783,7 @@ deck file kong2tf -s ./kong.yaml --generate-imports-for-control-plane-id "0dea9a
 ## Flags
 
 `-g`, `--generate-imports-for-control-plane-id`
-: `import` blocks will be added to Terraform to adopt existing resources. Typically used after `deck gateway dump --with-id` to obtain the IDs of all entities.
+:  Generate terraform import statements for the control plane ID. Typically used after `deck gateway dump --with-id` to obtain the IDs of all entities.
```

Comment:
fyi for reviewers: This change doesn't need to be version tagged, it's just an improvement to the explanation text.